docs/option/stateLoadCallback.xml

```diff
@@ -45,7 +45,7 @@ new DataTable('#myTable', {
 });
 ]]></example>
 
-	<example title="Load state from a server via Sjax (prior to 1.10.13)"><![CDATA[
+	<example title="Load state from a server via Ajax (prior to 1.10.13)"><![CDATA[
```

Comment:
This change is is actually incorrect - "Sjax" is correct in this case since the code uses `async: false` - therefore it is synchronous JavaScript And Xml". Unusual but correct.